README.md

```diff
@@ -41,33 +47,44 @@ Once the collection is installed, you can use it in a playbook by specifying the
     delegate_to: localhost
 ```
 
+## Optimizing Playbooks
```

Comment:
Is optimizing the right wording? Because the "optimization" is mostly execution time optimization which also comes at a cost. Should we rephrase to something in the trend of `decrease playbook execution times`

Reply:
Yeah, optimization can have multiple elements. One being playbook speed, others include reliability, maintainability ect.
I think the HTTPAPI also provides reliability in the form of using multiple APIC hosts and maintainability by allowing connection variables to move to the inventory. However, yes, the suppress options are mainly for speed at the expense of some functionality. I think optimization is still okay to be used and it also gives us a document to extend if anyone finds more optimizations other than speed.

Reply:
ok